### PR TITLE
Shoes 4 pre1 release post

### PR DESCRIPTION
**Pull request by PragTob**
A release post scheduled to release tomorrow along with the real release :) (wow release 3 times in one sentence, I hope the post is better :P)

I'd absolutely love a review @wasnotrice @jasonrclark + everyone else.

Please don't hesitate to suggest deleting a paragraph/section/whatever. I tend to write too much/too long posts - I'm working on it but it's not getting better :(

Also moving sections etc.

Jay and thank you!
Tobi

### CODE REVIEW

File: _posts/2014-05-10-shoes-4pre1-release.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+<div class='hero-unit'>
+  <h1>The Shoes Blog</h1>
+</div>
+<h2><a href="{{ post.url }}">shoes 4.pre1 released!</a></h2>
+<h3>2014-05-10 16:00</h3>
+
+Hello shoesers out there! Long time no see! Wondering what we have been up to?
+
+Well we've been hard at work working on [shoes 4](https://github
+.com/shoes/shoes4) - and today is a happy day because we can announce the
```
Comment (jasonrclark) [inline _posts/2014-05-10-shoes-4pre1-release.md]: Maybe just "hard at work on"?

File: _posts/2014-05-10-shoes-4pre1-release.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+first preview release!
+
+## shoes 4?
+
+shoes 4 is a complete rewrite of the shoes DSL, shoes 3 is great but it has its
```
Comment (jasonrclark) [inline _posts/2014-05-10-shoes-4pre1-release.md]: Should be a "." rather than ","

File: _posts/2014-05-10-shoes-4pre1-release.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+ problems... e.g. a lot of C-code which made it hard to maintain plus
+ installation problems that were hard to fix.
```
Comment (jasonrclark) [inline _posts/2014-05-10-shoes-4pre1-release.md]: Already said "hard" before... maybe "tricky" or "tough" or "difficult installation problems"?

File: _posts/2014-05-10-shoes-4pre1-release.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+JVM installed but don't worry - all you need to ever write and worry about
+is ruby, not Java.
+
+## Preview Release?
+
+shoes 4 is not quite done yet - this is the first release after almost 2 years
+ of work - it's not quite ready for production use and is missing some
+ features. However there is already a lot in there and a lot of vintage shoes
+  programs run just fine and it's definitely possible to enjoy the power of
+  shoes again!
```
Comment (jasonrclark) [inline _posts/2014-05-10-shoes-4pre1-release.md:46]: This paragraph the sentences run on a bit. Maybe split a couple of them at the "and"s?

File: _posts/2014-05-10-shoes-4pre1-release.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+* almost [600 resolved issues](https://github.com/shoes/shoes4/issues?state=closed)
+* over 2000 commits
+* many vintage shoes samples [are running](https://github
+.com/shoes/shoes4/blob/master/samples/README)
+* ability to install as a gem!
+* contributions from a lot of amazing people
+
+### What's missing?
+
+* proper handling of quite some styles/methods for the elements
```
Comment (jasonrclark) [inline _posts/2014-05-10-shoes-4pre1-release.md]: "quite some" => "quite a few"

File: _posts/2014-05-10-shoes-4pre1-release.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+* installers (right now it's "just" a gem)
+* packaging functionality - that's one of the big things missing a proof of
```
Comment (jasonrclark) [inline _posts/2014-05-10-shoes-4pre1-release.md]: "." after the "missing"?

File: _posts/2014-05-10-shoes-4pre1-release.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+
+Alright, alright - enough talking!
+
+At first you have to install [JRuby](http://jruby.org/) - preferably a
+version of the 1.7 series (most current release is 1.7.12). If you use rvm
+you can do it like this: `rvm install jruby`
+
+Then while JRuby is selected as your main ruby do:
+
+    gem instal shoes --pre
```
Comment (jasonrclark) [inline _posts/2014-05-10-shoes-4pre1-release.md]: "instal" => "install"

File: _posts/2014-05-10-shoes-4pre1-release.md [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+
+et voila!
+
+After installing the gem successfully you should have a `shoes` command
+available to you on the console for running shoes programs. E.g.
+
+    shoes path/to/my/program/my_app.rb
+
+If you want to install shoes 4 straight from the github resources there are
```
Comment (jasonrclark) [inline _posts/2014-05-10-shoes-4pre1-release.md]: "the github resources" => "github"?

**Comment by jasonrclark** [comment]
Apart from my minor edits, I like it. Doesn't seem overly long for what it is. You could probably play up the birthday aspect a little more if you wanted, but it reads well as it stands.

**Comment by davorb** [comment]
I also think it's nice that we're keeping it short and to the point. I think we might want to consider changing the heading, though. "2014-05-10 16:00" doesn't really feel like something that's making me want to read the rest of the paragraph.

**Comment by PragTob** [comment]
Thanks for your feedback tried to accommodate it :)

**Comment by wasnotrice** [comment]
@PragTob looks great. I made a pull request on your repo with some wording and punctuation changes...see what you think (also fixed @davorb's date formatting :)

The only other thing I might add is a code sample and set of screenshots showing shoes 4 and shoes 3 maybe?

**Comment by PragTob** [comment]
Screenshots and a code sample is good - I'd probably take the code sample from [my older blog post](http://pragtob.wordpress.com/2013/07/17/shoes-4-a-progress-report/) and add screenshots of the manual/tankspank.

With shoes3 do you mwan like having the screenshots side by side? Dunno if we need that or what do you mean :)

**Comment by PragTob** [comment]
A little sample code and screenshots of samples are in! =D

**Comment by wasnotrice** [comment]
Looks good! I think it's fine to not show shoes 3 screenshots. I was thinking it would demonstrate compatibility, but you're right probably unnecessary

**Comment by PragTob** [comment]
@wasnotrice yes as long as it demonstrates compatibility they'd basically be the same :) Maybe for a video or so which I'd like to make... if only I had the time :-) Or well rather would allocate time for that.

**Pull request merged**